packages/solo5-cross-aarch64/solo5-cross-aarch64.0.7.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "martin@lucina.net"
+authors: [
+  "Dan Williams <djwillia@us.ibm.com>"
+  "Martin Lucina <martin@lucina.net>"
+  "Ricardo Koller <kollerr@us.ibm.com>"
+]
+homepage: "https://github.com/solo5/solo5"
+bug-reports: "https://github.com/solo5/solo5/issues"
+license: "ISC"
+dev-repo: "git+https://github.com/solo5/solo5.git"
+build: [
+  ["env" "TARGET_CC=aarch64-linux-gnu-gcc" "TARGET_LD=aarch64-linux-gnu-ld" "TARGET_OBJCOPY=aarch64-linux-gnu-objcopy" "./configure.sh" "--prefix=%{prefix}%"]
+  [make "V=1"]
+]
+install: [make "V=1" "install-toolchain"]
+depends: [
+  "conf-pkg-config" {build & os = "linux"}
+  "conf-libseccomp" {build & os = "linux"}
+  "solo5" {= version}
+]
+depexts: [
+  ["linux-headers"] {os-distribution = "alpine"}
+  ["kernel-headers"] {os-distribution = "fedora"}
+  ["kernel-headers"] {os-distribution = "rhel"}
+  ["linux-libc-dev"] {os-family = "debian"}
+  ["gcc-aarch64-linux-gnu"] {os-family = "debian"}
+]
+available: [
+  (arch != "arm64") &
+  (arch != "arm32") &
+  (arch != "ppc64") &
+  (arch != "s390x") &
+  (os = "linux" & os-family = "debian")
```

Comment:
Is it really available only on Debian? If so why even add depexts for other distributions?

Reply:
I'm not the author of this choice and currently, `solo5-cross-aarch64` works on `ubuntu` and ArchLinux (/cc @TheLortex) but I think Martin Lucina was pretty conservative about this new package. As an experimental release (see the Changelog), I think we should keep these constraints and then check if everything works for others distributions :+1:.